ui/lib/types.ts

```diff
@@ -55,3 +55,10 @@ export interface Syncable {
   namespace?: string;
   clusterName?: string;
 }
+
+export interface MultiRequestError {
+  kind?: FluxObjectKind;
+  clusterName?: string;
+  namespace?: string;
+  message?: string;
+}
```

Comment:
Could we implement this as `ListError & { kind: FluxObjectKind; }` just to keep it sync'd up w/ the pb.ts files in case the BE response changes?